Gemfile.lock

```diff
@@ -120,8 +120,8 @@ GEM
     bullet (6.0.2)
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
-    byebug (11.1.1)
-    capistrano (3.12.1)
+    byebug (11.1.3)
+    capistrano (3.14.0)
```

Comment:
### [capistrano](https://capistranorb.com/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.12.1 → 3.14.0

[_[change-log](https://github.com/capistrano/capistrano/releases), [source-code](https://github.com/capistrano/capistrano)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/capistrano/capistrano/compare/v3.12.1...v3.14.0).

These are the individual commits:
- (19e5f8a) [Use env to find ssh path](https://github.com/capistrano/capistrano/commit/19e5f8a626e6b27eb1676f256fa53486c953fd7b)
- (81bdc59) [Merge pull request #2053 from JoeDupuis/master](https://github.com/capistrano/capistrano/commit/81bdc59ff2e38295b07bf580b4a1b8d9c402ba3b)
- (279a0af) [Remove CHANGELOG task from PR template (#2054)](https://github.com/capistrano/capistrano/commit/279a0afa3e3c01f8e3fdd9bc13881bd42e256e1b)
- (8f3aca4) [Release 3.13.0](https://github.com/capistrano/capistrano/commit/8f3aca476875b97cf2f2f2e28cb01e6e9508ee61)
- (50dd0ee) [Add prompt option for user input](https://github.com/capistrano/capistrano/commit/50dd0ee4f3f074a6b05152c1db7b83aab9d1898f)
- (46e00de) [Merge pull request #2055 from rolandasb/user-input-prompt-op…](https://github.com/capistrano/capistrano/commit/46e00dec1524bd07137681215aab10569d10b495)
- (3fbbbd8) [Update docs website icons, optimize images (#2056)](https://github.com/capistrano/capistrano/commit/3fbbbd8f5b9ab789729b93070393c881e76ab445)
- (118c923) [Fix CI builds for EOL Ruby versions (#2058)](https://github.com/capistrano/capistrano/commit/118c923705eeb658248e466ecc82c1b40e7ab251)
- (e136a3b) [Release 3.14.0](https://github.com/capistrano/capistrano/commit/e136a3bc02945ec0537946a2166d8688751c6d12)


</details>